src/functional.test.ts

```diff
@@ -640,6 +642,263 @@ describe('create-release-branch (functional)', () => {
       );
     });
 
+    it('updates the dependency version in package "b" when package "a" version is bumped', async () => {
```

Comment:
`create-release-branch` runs `yarn constraints --fix` and doesn't care what the constraints are exactly. So it isn't necessary the case that `create-release-branch` always updates dependencies like this — it really depends on what the project's constraints file does (a project may have different constraints, in theory). That said, I'm okay with using this test name since it points to why we decided to add `yarn constraints --fix` in the first place. If we ever want to run constraints for a different reason we may have to adjust this test name to be more accurate, but I don't anticipate that happening any time soon.